azure-iot-device/azure/iot/device/iothub/aio/async_clients.py

```diff
@@ -85,6 +90,46 @@ def _on_disconnected(self):
         self._inbox_manager.clear_all_method_requests()
         logger.info("Cleared all pending method requests due to disconnect")
 
+    async def _enable_feature(self, feature_name):
+        """Enable an Azure IoT Hub feature
```

Comment:
these enable and disable are not new methods only the else part is new right ?

Reply:
Actually there's nothing added at all - they're just moved to a different part of the file to better organize it

---
In reply to: [454010147](https://github.com/Azure/azure-iot-sdk-python/pull/620#discussion_r454010147) [](ancestors = 454010147)